components/prism-tt2.js

```diff
@@ -0,0 +1,78 @@
+(function(Prism) {
+
+	var tt2_pattern = /\[%[\s\S]+?%\]|^%%*?/;
+
+	Prism.languages.tt2 = Prism.languages.extend('markup', {
+		'tt2': {
+			pattern: tt2_pattern,
+			inside: {
+				'comment': {
+					pattern: /(^|[^\\$])#.*/,
+					lookbehind: true
+				},
+				'delimiter': {
+					pattern: /^(?:\[%|%%)-?|-?%]$/i,
+					alias: 'punctuation'
+				},
+				'string': /(["'])(?:\\.|(?!\1)[^\\\r\n])*\1/,
+				'number': /\b0x[\dA-Fa-f]+\b|(?:\b\d+\.?\d*|\B\.\d+)(?:[Ee][+-]?\d+)?/,
+				'keyword': /\b(?:GET|CALL|SET|DEFAULT|INSERT|INCLUDE|PROCESS|WRAPPER|BLOCK|IF|UNLESS|ELSIF|ELSE|SWITCH|CASE|FOREACH|WHILE|FILTER|USE|MACRO|RAWPERL|PERL|TRY|THROW|CATCH|FINAL|NEXT|LAST|RETURN|STOP|CLEAR|META|TAGS|DEBUG)\b/,
+				'operator': /\b(?:=>|==|!=|<|<=|>|>=|&&\|\|!|and|or|not)\b/,
```

Comment:
Something looks weird here: `&&\|\|!`. Is it intended? Wasn't it supposed to be something like... `&&|\|\||!` instead, maybe?

Also, I'm not sure the word-boundaries are a good idea for the operators other than words. Your regexp should probably look like:
`/=>|==|...|\b(?:and|or|not)\b/`

Reply:
Cut and paste errors from my side.